### ISSUE TITLE
[Bug]: use system; can not be executed successfully

### ISSUE THREAD

**Issue by goodMan-code** [matrixorigin/matrixone#6254]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

Problem ：
The last new MO，commit 5f330fb9a2535298e244a4df15ab33bde343fea8，
1. Start MO by : ./mo-service -launch ./etc/launch-tae-logservice/launch.toml
1. execute :  mysql -h 127.0.0.1 -P 6001 -udump -p111
2. execute :  use system;    this stmt can not be executed successfully.
3. I can execute  use system;  by  JDBC link MO,  maybe  problems of client or Initialize.
![use](https://user-images.githubusercontent.com/74952516/199876168-0665867c-dbf0-40ed-9d63-e93a2a7d177b.png)

**Comment by goodMan-code** [comment]
该语句通过 JDBC 连接 MO 是可以成功执行的，怀疑是客户端的问题。

**Comment by tianyahui-python** [comment]
[mo.log.tar.gz](https://github.com/matrixorigin/matrixone/files/9936518/mo.log.tar.gz)

**Comment by daviszhen** [comment]
i almost know the reason. it will be fixed.

**Comment by goodMan-code** [comment]
I have Recheck it, can be closed on commit commit 45c43c065b169c8dc00c9f8ed6964dac3e0a6ef8
